_contentTemplates/common/general-info.md

```diff
@@ -13,8 +13,6 @@ the `Format` is culture-specific and the same format may produce different resul
 #cdn
 You can reference the built-in Telerik assets from a cloud CDN instead of a local resource on your server.
 
->caption Reference the Telerik assets from the cloud CDN
```

Comment:
Be consistent with the use of captions - either use them for all code snippets or don't use them.

Reply:
This is also used in the Themes Overview article where it is consistent with the rest of the article https://docs.telerik.com/blazor-ui/themes/overview and the second place it is used is the What You Need article where the first two snippets are in bullets which is why they don't have captions. Should this still be removed?

Reply:
We generally have captions above all code snippets, figures and tables. The only exception are code snippets within bullet lists - to make those you already have to define a caption with the language name, so it does not make sense to add more captions (or maybe the parsing also didn;t work, I don't remember).